src/user-guide/interventions/index.rst

```diff
@@ -87,6 +87,19 @@ Re-Run a Task
 
          $ cylc trigger <workflow>//<task>
 
+   .. tab-item:: Explanation
+      :sync: explanation
```

Comment:
IMO this should not be a tab, which is meant to be for your preferred choice of interacting with Cylc. Instead it should be a "spoiler" admonition or ordinary admonition

Reply:
(It depends on whether we are trying to keep the "advanced" info somewhat hidden, to avoid scaring users with the most basic needs)

Reply:
Ronnie suggested a "spolier" which hides the content.

Reply:
Yep I like that.